lib/grammers-session/src/lib.rs

```diff
@@ -120,7 +120,7 @@ impl Session {
             .next()
     }
 
-    pub fn insert_dc(&self, id: i32, server_addr: SocketAddr, auth: [u8; 256]) {
+    pub fn insert_dc(&self, id: i32, addr: SocketAddr, auth: [u8; 256]) {
```

Comment:
Why?

Reply:
I think it's obvious which address is meant, also server_addr was aliased to addr later in the code. Should I rename it back to server_addr?

Reply:
Nah it's fine, I just wondered.